cmd/common.go

```diff
@@ -100,6 +100,25 @@ func getNullString(flags *pflag.FlagSet, key string) null.String {
 	return null.NewString(v, flags.Changed(key))
 }
 
+func getNullStrings(flags *pflag.FlagSet, key string) []null.String {
```

Comment:
As I [mentioned in the kafka PR](https://github.com/loadimpact/k6/pull/617/files/d62aad654a84347b2d318cb90a8c72fb98505ea0#r188242950), I don't think it's necessary it to have `null.String` slices.

Reply:
Yep, I'll get this updated